ShaTuApp/setup_ShaTuDB.sql

```diff
@@ -62,19 +189,243 @@ INSERT INTO Student (UserId, FirstName, LastName)
 INSERT INTO StudentModel (UserId, ScaffoldLevel)
    VALUES ('test@regis.edu', 'Extreme');
```

Comment:
There is no User Table anymore; suggest delte line 183

There is an error that throws due to ln 189-190: suggest delete and let them create the test user so it gets created correctly when account is created.